src/components/Registration/RegPeopleAndRoles.vue

```diff
@@ -236,6 +236,11 @@ export default class RegPeopleAndRoles extends Mixins(PeopleRolesMixin) {
       this.currentOrgPerson.deliveryAddress = cloneDeep(this.currentOrgPerson.mailingAddress)
     }
 
+    if (isOrganization) {
+      this.currentOrgPerson.isLookupBusiness = true
+      this.currentOrgPerson.showOptionalBN = false
```

Comment:
In Edit UI, Roy was OK with always showing the BN line but with "Not entered" when not available. Maybe this would make your life easier here?

Reply:
I am trying the same here. So when user try to add an org we default to lookup by setting `isLookupBusiness = true` and until user select a business, hide optional BN text field `showOptionalBN = false`

Reply:
I'm resolving this comment and I leave it up to you.